src/noun.substance

```diff
@@ -187,13 +187,26 @@ hs: air
 hypo: argonon
 hypo: chemical_element
 
+w: arenite
+d: a sedimentary rock with sand-sized grains and < 15% matrix material (< 30 µm),. The relative proportions of feldspar, quartz, and lithic fragments are not defined (Pettijohn et al., 1987; Tucker, 2001)
+hyper: arkose
+hyper: litharenite
+hyper: quartz_arenite
+hypo: sandstone[2]
+mt: noun.cognition:lithology
+
 w: Aristocort mu noun.communication:trade_name
 w: Aristopak mu noun.communication:trade_name
 w: Kenalog mu noun.communication:trade_name
 w: triamcinolone
 d: a synthetic corticosteroid (trade names Aristocort or Aristopak or Kenalog) used as an anti-inflammatory agent
 hypo: adrenal_cortical_steroid
 
+w: arkose
+d: an arenite with feldspar (> 25%) and quartz as major constituents. It can also contain lithic fragments. Alkali-feldspar dominates the total feldspar. Most arkoses are derived from granites or gneisses (Neuendorf et al., 2005; Tucker, 2001)
```

Comment:
glosses are not following https://simplyphilosophy.org/study/aristotles-definitions/, why?

Reply:
Just reviewed and adapted GeolBA's definitions to the Aristotelian way.